### PR TITLE
feat: Added Geo vehicles

### LINKED ISSUE #52: Add all Geo vehicles to common.js
Find all Geo vehicles in Appendix A of this document https://www.scca.com/downloads/49336-2020-solo-rules-complete-reduced-2020-03-16/download and add them to https://github.com/Bjorn248/scca_classifier/blob/master/common.js in a similar way to other Makes.

### PR DESCRIPTION
**Pull request by Llambi**
Resolves #52

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -2938,6 +2938,71 @@ const allSoloCars = {
...
+      'all': ['hs']
+    },
+    'Prizm':{
+      'all': ['hs']
+    },
+    'Spectrum':{
+      'all': ['hs']
+    },
+    'Storm':{
+      '2005': ['d'],
```
Comment (Bjorn248) [inline common.js:2952]: `d` is not a valid class

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Just need to fix invalid classes

File: common.js (hunk 1 above)
Comment (Llambi) [reply common.js:2952]: Sorry I misread the pdf 😢

File: common.js [hunk 2]
```diff
@@ -2938,6 +2938,71 @@ const allSoloCars = {
...
+    },
+    'Spectrum':{
+      'all': ['hs']
+    },
+    'Storm':{
+      '2005': ['sp'],
+      '2006': ['dsp'],
+      '2007': ['dsp'],
+    },
+    'Cobalt SS (N/A)':{
```
Comment (Bjorn248) [inline common.js:2956]: The Cobalt and Cobalt SS are Chevrolet models.

File: common.js [hunk 3]
```diff
@@ -2938,6 +2938,71 @@ const allSoloCars = {
...
+      '2005': ['dsp'],
+      '2006': ['dsp'],
+      '2007': ['dsp'],
+    },
+    'Cobalt SS Turbo':{
```
Comment (Bjorn248) [inline common.js:2961]: The Cobalt and Cobalt SS are Chevrolet models.

File: common.js [hunk 4]
```diff
@@ -2938,6 +2938,72 @@ const allSoloCars = {
...
+      '2008': ['dsp'],
+      '2009': ['dsp'],
+      '2010': ['dsp'],
+    },
+    'HHR SS Turbo':{
```
Comment (Bjorn248) [inline common.js:2966]: HHR is a Chevrolet Model

File: common.js [hunk 5]
```diff
@@ -2938,6 +2938,72 @@ const allSoloCars = {
...
+      'all': ['dsp']
+    },
+    'J Body (4-cyl Turbo, Quad 4 DOHC, & V6)':{
```
Comment (Bjorn248) [inline common.js:2969]: This is a GM platform

File: common.js [hunk 6]
```diff
@@ -2938,6 +2938,72 @@ const allSoloCars = {
...
+      'all': ['dsp']
+    },
+    'L Body (Quad 4 & V6)':{
```
Comment (Bjorn248) [inline common.js:2972]: This is a GM platform

File: common.js [hunk 7]
```diff
@@ -2938,6 +2938,72 @@ const allSoloCars = {
...
+      'all': ['dsp']
+    },
+    'N Body (4-cyl Turbo, Quad 4, & V6)':{
```
Comment (Bjorn248) [inline common.js:2975]: This is a GM platform

File: common.js [hunk 8]
```diff
@@ -2938,6 +2938,72 @@ const allSoloCars = {
...
+      '1989': ['dsp'],
+    },
+    'Storm GSi':{
+      '1985': ['dsp'],
+      '1986': ['dsp'],
+      '1987': ['dsp'],
+      '1988': ['dsp'],
+      '1989': ['dsp'],
+    },
+    'X Body (V6)':{
```
Comment (Bjorn248) [inline common.js:2992]: This is a GM platform

File: common.js [hunk 9]
```diff
@@ -2938,6 +2938,72 @@ const allSoloCars = {
...
+      'all': ['dsp']
+    },
+    'Cobalt SS Supercharged':{
```
Comment (Bjorn248) [inline common.js:2995]: This is a Chevrolet

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Seems like there are GM and Chevrolet models throughout the Geo section

File: common.js (hunk 1 above)
Comment (Bjorn248) [reply common.js]: No problem, the PDF is a bit confusing

File: common.js (hunk 2 above)
Comment (Llambi) [reply common.js]: Sorry I misread the pdf

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**